File: orig/hhh-2008-03-29/Food.cc
#include "Food.h"

class Food;

Food::Food(char *filename, int x, int y, int speed,AnimationFactory* af):
	Ghost1(filename, x, y, speed, af){
	this->alive = true;
};

Food::~Food(){
//	int i;
//	for(i=0; i<frames.size(); i++){
//		SDL_FreeSurface(frames[i]);
//	}
};

File: food.go
package main

type foodinfo struct {
	sx, sy, speed int
}

type Food struct {
	*Ghost1
	alive bool
}

func newFood(filename string, x, y, speed int, af *AnimationFactory) *Food {
	return &Food{
		Ghost1: newGhost1(filename, x, y, speed, af),
		alive:  true,
	}
}